CHANGELOG.md

```diff
@@ -4,6 +4,80 @@
 
 ## DFX
 
+### feat: dfx extension can now be installed from external registries
+
+Modifies the `dfx extension install` command to take a flag `--registry` that allows installing extensions from external registries. Here is an example:
```

Comment:
Suggestion:

The `dfx extension install` command now supports `--registry <url to registry json>` for installing an extension from an external registry. Example usage:

```bash
dfx extension install your_extension --registry https://raw.githubusercontent.com/your-org/your-repo/master/dfx-extensions-registry.json
```

For more information about hosting external registries, please see https://github.com/dfinity/dfx-extensions/docs/hosting-external-registries.md